extensions-rhel-9.0.yaml

```diff
@@ -4,6 +4,8 @@
 
 repos:
   - rhel-9-nfv
+  # Temporarily included for kata-containers
+  - rhel-8-server-ose
```

Comment:
On this topic, one thing we should probably support in rpm-ostree is to have repos mentioned in `repo-packages` be enabled but used only for the mentioned packages. This would make things like this cleaner, so that we don't accidentally also ship other things from that repo that we didn't intend because of EVR ordering funkiness.

Reply:
Agree. Right now I'm doing that manually in the repo configs: https://github.com/openshift/os/pull/937/files#diff-cf4ac67e6195e36ef10dd53e72af30a0346517d3faf3857f88e74204c7373e3bR94